test/dlc_tests/eks/mxnet/training/test_eks_mxnet_multinode_training.py

```diff
@@ -171,7 +162,10 @@ def _run_eks_multi_node_training_mpijob(namespace, job_name, remote_yaml_file_pa
     Function to run eks multinode training MPI job
     """
 
-    run(f"kubectl create namespace {namespace}")
+    does_namespace_exist = run(f"kubectl get namespace | grep {namespace}", warn=True)
```

Comment:
since this should be a bool, and you're using warn=True, I would check the `return_code` attribute is 0

Reply:
Also want to check - is grep "namespace" enough? are there any namespaces that are substrings of others?

Reply:
This should be enough. No substrings are used to verify the namespace